packages/edit-post/src/plugins/copy-content-menu-item/index.js

```diff
@@ -8,27 +8,29 @@ import { withState, compose } from '@wordpress/compose';
 
 function CopyContentMenuItem( { createNotice, editedPostContent, hasCopied, setState } ) {
 	return (
-		<ClipboardButton
-			text={ editedPostContent }
-			role="menuitem"
-			className="components-menu-item__button"
-			onCopy={ () => {
-				setState( { hasCopied: true } );
-				createNotice(
-					'info',
-					'All content copied.',
-					{
-						isDismissible: true,
-						type: 'snackbar',
-					}
-				);
-			} }
-			onFinishCopy={ () => setState( { hasCopied: false } ) }
-		>
-			{ hasCopied ?
-				__( 'Copied!' ) :
-				__( 'Copy All Content' ) }
-		</ClipboardButton>
+		editedPostContent.length > 0 && (
```

Comment:
I had some worries that editedPostContent may be null or undefined, but I checked the docs of the selector that ultimately is used, and it refers a string is always returned  `* @return {string} Post content.`